bc_obps/registration/tests/endpoints/v1/_users/test_user_id.py

```diff
@@ -36,8 +36,9 @@ def test_users_endpoint_unauthorized_roles_cannot_get(self):
         assert response.status_code == 401
 
     def test_get_user_with_invalid_user_id(self):
+        TestUtils.authorize_current_user_as_operator_user(self, operator_baker())
         response = TestUtils.mock_get_with_auth_role(
-            self, endpoint=custom_reverse_lazy("get_user", kwargs={"user_id": '99999'}), role_name="cas_admin"
+            self, endpoint=custom_reverse_lazy("get_user", kwargs={"user_id": '99999'}), role_name="industry_user"
```

Comment:
Why the switch in roles? cas_admins can still get users, right?

Reply:
This endpoint is currently restricted to industry admin users only. This may change in the future. Previously, data validation occurred before user validation, but now user validation takes precedence over data validation.

Reply:
Right, I'm mixing this up with cas's user-operator endpoints. Thanks!